Update kicker design for action front cards

marjisound opened the pull request:
<!-- In this repo you can label a PR with the "PR Deployment" label to deploy the code to a publicly accessible url -->
## What does this change?
This PR updates the design for actionable card where user can take an action by opening the article, e.g. filling in a callout form.

The kicker font size and its position in the card is related to the heading font size, and it has to be very accurate because these cards could be located next to a non-actionable card, therefore, the headings of these 2 cards should be **aligned**.

Also there has to be support for long kickers where it ends up in multiple lines. But obviously, when the kickers are multi-line, the card heading could be dis-aligned from the heading of the neighbouring card. This is acceptable as it's a rare case.

Also, in bigger cards, when the kicker is multi-line there could be a gap between the 2 lines. This is also acceptable (discussed with @benwuersching )

## Why?
Currently, there is not a very clear way of visually setting apart what is a "callout" article on fronts. Because of this, some of our users currently think callout articles are actual articles. This ambiguity could be having a negative impact on our users' experience. Additionally, we think that if we can make it clearer to our users that an article is a callout, we could drive user engagement in our callout campaigns.

<img width="965" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/15894063/f5d7989c-33ac-4e28-94ff-a2811cad239d">

## Screenshots

| Before      | 
|-------------|
| ![image](https://user-images.githubusercontent.com/15894063/233136352-8d121228-5aca-4390-bcbf-69fa73dec764.png) | 

| After      |
|------------|
| ![image](https://user-images.githubusercontent.com/15894063/233136090-7ee7ed5e-f3d8-46a5-babd-c0f5c2f1a353.png) |


| Long kicker Before      | 
|-------------|
| ![image](https://user-images.githubusercontent.com/15894063/233138625-74dd2668-0ac8-47a9-858d-c55c3d0b98db.png) | 

| Long kicker After      |
|------------|
| ![image](https://user-images.githubusercontent.com/15894063/233138824-a5ad2660-0aed-4df5-a10e-ac10ef5a3d53.png) |

[before]: https://example.com/before.png
[after]: https://example.com/after.png

<!--
You can add extra rows by repeating the last row in the table and then using new unique labels. E.g.

| ![before2][] | ![after2][] |

You can then reference the labels and map them to corresponding links.

[before2]: https://example.com/before2.png
[after2]: https://example.com/after2.png
-->


<!--
## Unexplained Chromatic diffs

We use Chromatic for visual regression testing on our Storybook stories. It's
generally pretty good, but it sometimes gives 'false positives' -- it seems to
detect a change in a component which hasn't changed, or which hasn't been
affected by the code in your PR.

If you've looked at the Chromatic diffs and can't see any connection to your
code, please reach out to a member of the Web Experiences team, who will be able
to advise. It would also be helpful to add the false positive to our
[ongoing log of false positives](https://docs.google.com/spreadsheets/d/1FvItNTMFXIpI4rCrZ4mQ0CRouT06sSVro168f6oKPm4/edit?usp=drive_open&ouid=117150399571694275917#gid=0).
-->

marjisound commented:
![](https://github.trello.services/images/mini-trello-icon.png) [COMMUNITY: Promote callouts better on fronts (re-designing the card for when it is a callout rather than an article)](https://trello.com/c/eDBN0hxf/842-community-promote-callouts-better-on-fronts-re-designing-the-card-for-when-it-is-a-callout-rather-than-an-article)

paperboyo commented:
This is a loose comment about the above design for kickers only (the fact that they have a background colour in _after_ screenshots) and not about Tell us CTAs.

I wonder if adding a background to free-text kickers (added manually to stories by editors) as opposed to having them only for design/tone tags (if that’s a correct term), Analysis, Explainer, Callout etc, will dilute the recently-gained increased clarity regarding article tones? In other words: my hunch is, having only as many kickers as there are tones having a background and leaving free-text kickers as-is (without backround) makes the article tone (type) more prominent. Maybe even as prominent as not needing a specific Tell us CTAs (?).

Imagine: these design tags make article tones much more clear (yay!):

![image](https://user-images.githubusercontent.com/6032869/233840137-a53a1529-5cee-42bb-9300-5c4d51668204.png)
![image](https://user-images.githubusercontent.com/6032869/233840169-e6eca3cc-c4bd-4a7b-8027-bae80ceb624b.png)

We just need to expand them to all tones (including `tone/callout`).

Using the same design for free-text kickers goes against this clarity, right? It would be like reusing this design at the top of every article for series and for keyword tags too. Tones would drown in them again.

IMHO and all. And also based solely on the screenshots in this PR, so I may be simply mistaken. Happy to discuss and explain myself.

cc @HarryFischer @benwuersching

marjisound commented:
Hi @paperboyo Thanks for your comment. I think Ben and you already discussed this. So for now we can move forward with this change and will chat more about expanding it in the future :)

benwuersching commented:
Yes, @paperboyo and I had a chat about it which clarified things.

rhiannareechaye commented:
Hey @marjisound my brain isn't used to seeing cards like this in a row with so much intro text - would you mind sharing a screenshot of it without the text (so just the kicker, heading and button) - would just be handy to see how it would look in most cases?

marjisound commented:
> Hey @marjisound my brain isn't used to seeing cards like this in a row with so much intro text - would you mind sharing a screenshot of it without the text (so just the kicker, heading and button) - would just be handy to see how it would look in most cases?

I just added a screen shot of how it'd look with the real data. We also added an AB test in the front end version of this re-design [here](https://github.com/guardian/frontend/pull/26078) So this re-design will be available for only 20% of the users. 
We will hold off, on the DCR version until we get the result of the AB test
